src/data/equiv/tree.lean

```diff
@@ -0,0 +1,138 @@
+
+/-
+Copyright (c) 2019 E.W.Ayers. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: E.W.Ayers.
+
+Prove that branching recursive datastructures are encodable by writing a little verified lisp parser.
+-/
+import data.equiv.encodable data.equiv.list data.tree
```

Comment:
Mathlib style would use a lot more empty lines. An empty line above the `import` line, and below it. Empty line after `namespace`, idem after `section`, basically every declaration is surrounded by empty lines.